cmd/prometheus/main.go

```diff
@@ -293,8 +293,8 @@ func main() {
 	a.Flag("config.file", "Prometheus configuration file path.").
 		Default("prometheus.yml").StringVar(&cfg.configFile)
 
-	a.Flag("web.listen-address", "Address to listen on for UI, API, and telemetry.").
-		Default("0.0.0.0:9090").StringVar(&cfg.web.ListenAddress)
+	a.Flag("web.listen-address", "Address to listen on for UI, API, and telemetry. Can be repeated.").
```

Comment:
maybe we can explicitly mention in `web.max-connections` desc that the limit applies to all listen address.


Also, I don't know if/how to state that for global/external URL building, the first one in the list may be used.